scripts/validate_docstrings.py

```diff
@@ -455,6 +455,10 @@ def validate_one(func_name):
             if not rel_desc:
                 errs.append('Missing description for '
                             'See Also "{}" reference'.format(rel_name))
+
+    if "\t" in doc.raw_doc:
```

Comment:
Can you make this check for tabs only at the start of lines? We may have cases where we include the literal `\t` in the docstring (e.g. `read_csv`).